website/js/components/insilicopred/SplicingLevelSubtile.js

```diff
@@ -0,0 +1,455 @@
+                        <td>{data.wild.zScore.toFixed(2)}</td>
+                        { isDeNovo &&  <td>{data.wild.donorPosition || 'n/a'}</td> }
+                    </tr>
+
+                    <tr>
+                        <td>Variant</td>
+                        <td className="sequence">{boldedDiff(data.variant.sequence, data.wild.sequence)}</td>
+                        <td>{data.variant.MES.toFixed(2)}</td>
+                        <td className={`pathos-prob-label-${variantRating}`} style={{fontWeight: 'bold'}}>
+                        {data.variant.zScore.toFixed(2)}
+                        </td>
+                        { isDeNovo &&  <td>{data.variant.donorPosition || 'n/a'}</td> }
+                    </tr>
+
+                    {
+                        isDeNovo && (
+                            <tr>
+                                <td>Reference Donor</td>
+                                <td className="sequence">{data.closest.sequence}</td>
+                                <td>{data.closest.MES.toFixed(2)}</td>
+                                <td>{data.closest.zScore.toFixed(2)}</td>
+                                <td>{data.closest.donorPosition}</td>
+                            </tr>
+                        )
+                    }
+                </tbody>
+            </table>
+        );
+    }
+}
+
+class SpliceSiteImpactTable extends React.Component {
+    render() {
+        const {prior, type, variantZScore, data} = this.props;
+
+        let extraMessage = null;
+
+        if (prior === 0.34) {
+            if (type === 'donor' && variantZScore < -2.0) {
+                /* special case #2: native donor (when type == 'donor') */
+                extraMessage = "This variant was classified as Moderate because the Z-score for the wild type is already low (less than -2) and the change due to variation is not considered large enough to warrant Severe.";
+            }
+            else if (type === 'acceptor' && variantZScore < -1.5) {
+                /* special case #3: native acceptor (when type == 'acceptor') */
+                extraMessage = "This variant was classified as Moderate because the Z-score for the wild type is already low (less than -1.5) and the change due to variation is not considered large enough to warrant Severe.";
+            }
+        }
+
+        return (<div>
+            <div style={{textAlign: 'center', fontWeight: 'bold', margin: '20px'}}>
+                Probability of Pathogenicity<br />
+                [Due to Splice {capitalize(type)} Damage]
+            </div>
+
+            <table className="splicing-level-table">
+                <tr>
+                    <td width="60%">Qualitative Category</td>
+                    <td width="30%">Z-score Range</td>
+                    <td width="20%">Probability</td>
+                </tr>
+
+                {
+                    // FIXME: highlight row that matches the prior prob value
```

Comment:
now or later?

Reply:
Forgot to remove that; it's been implemented two lines below, where it checks if the row's probability matches the prior prob., and also validates an optional check condition.

Expanding on the "check", there is one case, prob == 0.04, which has two matching rows; the check function defined for those rows disambiguates which one should be highlighted.